src/program/services/downloaders/__init__.py

```
@@ -369,15 +369,16 @@ def update_item_attributes(
-                except Exception as e:
+                except Exception:
                     continue
 
                 if isinstance(item, (Show, Season, Episode)):
-                    if not file_data.episodes:
-                        continue
-                    elif 0 in file_data.episodes and len(file_data.episodes) == 1:
-                        continue
-                    elif file_data.seasons and file_data.seasons[0] == 0:
+                    # Allow files with episodes OR files with dates (for daily shows)
+                    if not file_data.episodes and not file_data.date or (
```

Ruff (0.14.13)

Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)